libraries/http/source/Captures.ts

```diff
@@ -55,3 +55,4 @@ function regexpUnescape (text: string): string {
 
 const CAPTURE = /\\\$\\{\\{\s*(?<name>\S{0,32})\s*\\}\\}/g
 const SUBSTITUTE = /\${{\s*(?<name>\S{0,32})\s*}}/g
+const PADDING = '(?:^|\\s+)'
```

Comment:
NIT: technically it should be [lookbehind](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Lookbehind_assertion) rather than non-capturing group. But both will do.